### ISSUE TITLE
Date type support

### ISSUE THREAD

**Issue by ert78gb** [vlio20/angular-datepicker#198]
Hi,

Why support only moment and string input / output type?

thx,
Robi

**Comment by vlio20** [comment]
This is on my list of features to support native Date type as well. I will mark this issue as a FeatureRequest. If you want to PR it would be awesome!

**Comment by ert78gb** [comment]
Cool. I will need it in the next 2 week, so I will working on it...

**Comment by vlio20** [comment]
@ert78gb, any progress?

**Comment by ert78gb** [comment]
Not yet. The string and moment version is enough for me now

**Comment by vlio20** [comment]
cool, I will handle it than :)

Thanks

**Comment by Franklin89** [comment]
Hi @vlio20 
Great library 👍 
I am just wondering how this feature is coming along and if you have any timeline on it? our data structure uses the **Date** type. It works straight out of the box more ore less. Except that if we load the data and present it the ISO String is displayed instead of the format that was set at config. Selecting a new date afterwards works great.
If any help is needed I might be able to do something. Let me know thanks.
Franklin

**Comment by vlio20** [comment]
Unfortunately I don't have a lot of time developing this feature, so not much (any) progress. PR would be great.

**Comment by Franklin89** [comment]
Okay I'll try my best.
Any hint where to start?

**Comment by vlio20** [comment]
Basically, you have a utility service which does all the conversions.
Internally the module works only with Moment object.

**Comment by WaterBleu** [comment]
yeah ideally I can just have the ngModel bind to a date object that would be nice.
is there a new timeline for this? @vlio20
